CHANGELOG.md

```diff
@@ -4,6 +4,29 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## Unreleased
+### Added 
+- Bump sensu-plugin runtime dep to 3.0 with Sensu Go to Sensu Core 1.x migration option (@jspaleta)
+- Added 2.5.1 ruby support (@jspaleta)
+- Added --verbose cmdline switch to aid in local testing
+- Added --disable_update cmdline switch to aid in local testing
+
+### Breaking Changes
+- Dropped 2.0 ruby support (@jspaleta)
```

Comment:
really we are dropping for ruby versons `> 2 && <= 2.3` for simplicity sake I ussually say drop support for ruby `< 2.3` but thats a matter of preference I think.